integration-tests/contract-multisig-set-and-remove-delegate.spec.ts

```diff
@@ -1,13 +1,22 @@
 import { CONFIGS } from "./config";
-import { MANAGER_LAMBDA } from "@taquito/taquito";
+import { MANAGER_LAMBDA, TezosToolkit  } from "@taquito/taquito";
 import { genericMultisig } from "./data/multisig";
 
 CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
-  const Tezos = lib;
- 
+  const Funder = lib;
```

Comment:
maybe add a not for why its called Funder? difference from the norm of other tests

Reply:
it's called Funder since it isn't actually signing anything. There are a few flextesa accounts that is provided by the sandbox. In this case, we only use the `flextesa_bootstrap` account to 'fund' the signer account we're using. 

It's purpose is to only fund the account to the signer TezosToolkit instance, which is still named `Tezos`